.gitignore

```diff
@@ -0,0 +1,2 @@
+builddir/
```

Comment:
Please drop the `.gitignore`. You can add a local one, meaning it looks like this
```gitignore
.gitignore
builddir/
.vscode/
```